runtime/cere-dev/src/weights/mod.rs

```diff
@@ -1,2 +1,3 @@
 pub mod ismp_grandpa;
+pub mod pallet_balances_balances;
```

Comment:
Shouldn't it be `pallet_balances` ?

Reply:
We can. I followed the same naming convention used in the Polkadot SDK, as seen here:
https://github.com/paritytech/polkadot-sdk/blob/polkadot-stable2412/polkadot/runtime/rococo/src/weights/pallet_balances_balances.rs